kas/common-kirkstone.yaml

```diff
@@ -61,8 +62,8 @@ repos:
     url: https://git.yoctoproject.org/meta-spdxscanner
     refspec: kirkstone
   meta-leda:
-    url: "https://github.com/eclipse-leda/meta-leda"
-    refspec: main
+    url: "https://github.com/SoftwareDefinedVehicle/meta-leda-fork"
```

Comment:
on the fork

Reply:
This is an old, commit. Fixed now. Check: https://github.com/eclipse-leda/leda-distro/pull/144/files for latest comparison